### ISSUE TITLE
Question: what's the reason of "no hypothesis. Finish decoding."

### ISSUE THREAD

**Issue by gandolfxu** [espnet/espnet#1371]
The log is pasted in the following. All the decoding results are the same. 
Why?

2019-11-18 19:28:49,359 (asr_utils:407) INFO: reading a config file from exp/train_rnnlm_pyto    rch_lm/model.json
 13 2019-11-18 19:28:50,464 (asr:691) INFO: (1/896) decoding BAC009S0724W0121
 14 2019-11-18 19:28:50,835 (e2e_asr_transformer:245) INFO: input lengths: 105
 15 2019-11-18 19:28:50,835 (e2e_asr_transformer:261) INFO: max output length: 105
 16 2019-11-18 19:28:50,836 (e2e_asr_transformer:262) INFO: min output length: 0
 17 2019-11-18 19:30:25,968 (e2e_asr_transformer:354) INFO: adding <eos> in the last postion in t    he loop
 18 2019-11-18 19:30:25,968 (e2e_asr_transformer:384) INFO: no hypothesis. Finish decoding.
 19 2019-11-18 19:30:25,969 (e2e_asr_transformer:405) INFO: total log probability: -1091.28392410    27832
 20 2019-11-18 19:30:25,969 (e2e_asr_transformer:406) INFO: normalized log probability: -10.59498    9554395953
 21 2019-11-18 19:30:25,969 (asr_utils:594) INFO: groundtruth: 广 州 市 房 地 产 中 介 协 会 分 >    析
 22 2019-11-18 19:30:25,969 (asr_utils:595) INFO: prediction : 我也是一个人一个人一个人一个人一个
    人一个人一个人一个人一个人一个人一个人一个人的人不是一个人的一个人在一起的人不是我的一个人在>    一起的人不是一个人的一个人我是不是一个人的人我是不是一个人的人不是我的一个人<eos>

**Comment by sw005320** [comment]
Could you give us more information? What kind of data are you using? What configurations?

**Comment by gandolfxu** [comment]
@sw005320 
I'm runing the example egs/aishell/asr1

**Comment by sw005320** [comment]
This would happen when all the hypotheses reach the end of the sentence. This is normal behavior.

**Comment by gandolfxu** [comment]
@sw005320 
But all the decoding results meet the same problem. I think that there may be some errors.

**Comment by sw005320** [comment]
Could you share your decoding config?
You seem to set something wrong.

**Comment by gandolfxu** [comment]
I used the default config.

asr_recog.py --config conf/decode.yaml --ngpu 0 --backend pytorch --batchsize 0 --recog-json dump/test/deltafalse/split16utt/data.1.json      --result-label exp/train_sp_pytorch_train/decode_test_decode_lm/data.1.json --model exp/train_sp_pytorch_train/results/model.last10.avg.b     est --rnnlm exp/train_rnnlm_pytorch_lm/rnnlm.model.best

conf/decode.yaml
batchsize: 0
beam-size: 10
penalty: 0.0
maxlenratio: 0.0
minlenratio: 0.0
ctc-weight: 0.5
lm-weight: 0.7

**Comment by sw005320** [comment]
I'm expecting your CTC part is not working.
Please check the warp CTC installation and/or CTC training loss.

**Comment by gandolfxu** [comment]
In the log file "exp/train_sp_pytorch_train/results/log",
"main/loss_ctc": 0.0

**Comment by sw005320** [comment]
OK, it seems that CTC is not working. Please make your CTC work or set the `maxlenratio` and `minlenratio` appropriately.
